cmd/vcert/utils.go

```diff
@@ -184,11 +185,11 @@ func fillCertificateRequest(req *certificate.Request, cf *commandFlags) *certifi
 			switch option {
 
 			case "m":
-				issuerHint = "MICROSOFT"
+				issuerHint = util.MicrosoftStr
```

Comment:
If the other two are using the "const" postfix, maybe microsoft should have it as well: MicrosoftConst.
Also I would expect this constants to be all upper case and have a better name to understand what they are referring to:
ISSUER_MICROSOFT
ISSUER_DIGICERT
ISSUER_ENTRUST

Reply:
yes you are right on the "const" post fix, but for the name I am following the name convention we have for this project that is just the name in camel case format, as you can see it on cmd/vcert/utils.go, in that place there is some constants and there have the camel case format.

Reply:
Fixed on most recent commit